### PR TITLE
+ first attempt for backend status codes

### PR DESCRIPTION
**Pull request by danielgomezrico**
The idea is to define some default status codes for some actions so we can use the same on all the projects.

### CODE REVIEW

**Comment by sarcilav** [comment]
I'm still not sure, of the social login error yielding 401, maybe 422 Unprocessable Entity 424 Failed Dependency and 428 Precondition Required are better options

**Comment by sarcilav** [comment]
I think that a middle ground is

```
403 Forbidden
The request was a valid request, but the server is refusing to respond to it. 
403 error semantically means "unauthorized", i.e. 
the user does not have the necessary permissions for the resource.
```

**Comment by danielgomezrico** [comment]
I like 424.

File: backend/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Backend styles
+
+Http Status Codes
+--------------
+
+Status codes for common escenarious:
+
+* Social Login:
+	- `424 Failed Dependency`: If the user dont exist on the db.
```
Comment (sarcilav) [inline backend/README.md]: This 424, needs a better description. Something like if the social access token is for a valid user but the user doesn't exist on the db

**Comment by danielgomezrico** [comment]
@sarcilav also added 401 for cases where the social token is invalid.

**Pull request merged**